src/benchmarks/real-world/ILLink/SampleProject/HelloWorld.template.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<OutputType>exe</OutputType>
-		<TargetFramework>net8.0</TargetFramework>
+		<TargetFramework>net9.0</TargetFramework>
```

Comment:
I think this should be changed to use `$(PERFLAB_TARGET_FRAMEWORKS)` and the `Condition`. For consistency sake.

Reply:
sure